modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/utils/MdcUtils.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.deployment.utils;
+
+import java.util.UUID;
+import org.eclipse.xpanse.modules.logging.LoggingKeyConstant;
+import org.eclipse.xpanse.modules.orchestrator.deployment.DeployTask;
+import org.slf4j.MDC;
+
+/** The tools of MDC. */
+public final class MdcUtils {
+
+    /** Put serviceId to MDC. */
+    public static void putServiceId(String serviceId) {
+        MDC.put(LoggingKeyConstant.SERVICE_ID, serviceId);
+    }
+
+    /** Put orderId to MDC And Set orderId for DeployTask. */
+    public static void putOrderId(String orderId, DeployTask task) {
```

Comment:
Not the right place to handle deployTask. 
MDCUtils must only set MDC based data.
Also, is it not possible to set the MDC directly in the storage classes when the entity is created?

And for service Id also do the same for deploy use case?